crankshaft-console/src/main.rs

```diff
@@ -1,2 +1,110 @@
-//! The Binary of the console TUI
-fn main() {}
+//! The main binary for the console TUI.
+
+/// The `conn` module handles connections to the crankshaft server.
+mod conn;
+/// The `input` module handles keyboard input.
+mod input;
+/// The `state` module manages the application state.
+mod state;
+/// The `term` module handles terminal initialization and cleanup.
+mod term;
+/// The `view` module renders the user interface.
+mod view;
+
+use Event::*;
+use KeyCode::*;
+use color_eyre::Section;
+use color_eyre::SectionExt;
+use color_eyre::eyre::eyre;
+use conn::Connection;
+use crossterm::event::Event;
+use crossterm::event::KeyCode;
+use crossterm::event::KeyEvent;
+use futures_util::StreamExt;
+use ratatui::layout::Constraint;
+use ratatui::layout::Direction;
+use ratatui::layout::Layout;
+use ratatui::text::Line;
+use ratatui::text::Span;
+use ratatui::widgets::Paragraph;
+use ratatui::widgets::Wrap;
+use state::State;
+use term::init_crossterm;
+use tonic::transport::Uri;
+use view::View;
+use view::styles::Styles;
+
+use crate::view::bold;
+
+/// The main function.
+#[tokio::main]
+async fn main() -> color_eyre::Result<()> {
+    let (mut terminal, _cleanup) = init_crossterm().unwrap();
```

Comment:
Could we not `unwrap` here and instead return the error with some context about trying to initialize the terminal?